### DISCUSSION TITLE
Discussing about react-select-search

### DISCUSSION THREAD

**Discussion by fhrryDeveloper** [npm/feedback#489] [CLI] [answered]
I am using react-select-search now.
How to get value in onchange function?

**Comment by MylesBorins** [comment, marked as answer]
Hey @fhrryDeveloper. This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
